<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="status">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Running</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Idle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Error</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="operator_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Operator Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="machine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);
</string>
    </property>
    <property name="text">
     <string>Machine ID :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Output :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>821</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins,Segoe UI,Roboto</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    color: white;
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:0,
        stop:0 #2ecc71, stop:1 #27ae60
    );
    font-family: &quot;Poppins&quot;, &quot;Segoe UI&quot;, &quot;Roboto&quot;;
    border-radius: 10px;
    padding: 10px 20px;
    letter-spacing: 1px;
}</string>
    </property>
    <property name="text">
     <string>Production Line Logger</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_machine">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_operator">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_output">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
